readme.md

```diff
@@ -370,6 +370,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [MQTT](https://github.com/hobbyquaker/awesome-mqtt) *(an "Internet of Things" connectivity protocol)*
 - [Hacking Spots](https://github.com/diasdavid/awesome-hacking-spots)
 - [For Girls](https://github.com/cristianoliveira/awesome4girls)
+- [OKR](https://github.com/domenicosolazzo/awesome-okr)
```

Comment:
Can you add a short and succinct description? See above for example on how it's done.